src/topp/SageAdapter.cpp

```diff
@@ -90,6 +96,598 @@ class TOPPSageAdapter :
   {
   }
 
+
+  struct RateMassPair
```

Comment:
indentation,
add short comment/doc what this struct is about

Reply:
Done!